source/clear-linux/tutorials/wordpress/web-server-install.rst

```diff
@@ -1,15 +1,16 @@
 .. _web-server-install:
 
-Create a Clear Linux\* based web server
-#######################################
+Create a |CL-ATTR| based web server
+###################################
 
-This tutorial shows you how to create a LAMP server using |CLOSIA| and how to use phpMyAdmin\* to manage an associated database.
+This tutorial shows you how to create a :abbr:`LAMP (Linux, Apache, MySQL, PHP)` server 
+using |CL-ATTR| and how to use phpMyAdmin\* to manage an associated database.
```

Comment:
This tutorial explains how to use |CL-ATTR| to create an Apache web server and use phpMyAdmin to manage an associated database, following a typical :abbr:`LAMP (Linux, Apache, MySQL, PHP)` stack model.